marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/opticPlanExportService.sjs

```diff
@@ -0,0 +1,85 @@
+/*
+  Copyright (c) 2020 MarkLogic Corporation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+'use strict';
+
+const op = require('/MarkLogic/optic');
+const search = require('/MarkLogic/appservices/search/search');
+const options = require("/data-hub/5/impl/hub-utils/invoke-search-options.sjs");
+
+const returnFlags = '<return-aggregates xmlns="http://marklogic.com/appservices/search">false</return-aggregates>\n' +
```

Comment:
Pinging @ryanjdew - should this be in XQuery instead of SJS? I ask that because we're dealing with a lot of XML here, and that's of course more difficult to do in SJS vs XQuery.

Reply:
@rjrudin I'm fine with having this in JavaScript. I recommended to @akshaysonvane that we call XSLT from JavaScript. @akshaysonvane, for readability, can we adjust this to use template literals for multiline string? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Sure, @ryanjdew.